### ISSUE TITLE
Anisotropic viscosity: `CS%KH_aniso = 2 k_a`

### ISSUE THREAD

**Issue by NoraLoose** [NOAA-GFDL/MOM6#1368]
There is an inconsistency between 
* how anisotropic viscosity is implemented in `MOM_hor_visc` and
* what is described in the [documentation](https://mom6.readthedocs.io/en/dev-gfdl/api/generated/modules/mom_hor_visc.html#horizontal-viscosity-in-mom).

The two would be consistent if `CS%KH_aniso` is interpreted as `2 k_a`. For instance, substituting this line
https://github.com/NOAA-GFDL/MOM6/blob/b92d763ec91d57d3868162d5d9a24827d2e53674/src/parameterizations/lateral/MOM_hor_visc.F90#L947
by
```
Kh(i,j) = Kh(i,j) + 2 k_a * (1. - CS%n1n2_h(i,j)**2) 
```
would be consistent with the documentation and with Smith & McWilliams, 2003. We also need `CS%KH_aniso = 2 k_a` [here](https://github.com/NOAA-GFDL/MOM6/blob/b92d763ec91d57d3868162d5d9a24827d2e53674/src/parameterizations/lateral/MOM_hor_visc.F90#L1003), [here](https://github.com/NOAA-GFDL/MOM6/blob/b92d763ec91d57d3868162d5d9a24827d2e53674/src/parameterizations/lateral/MOM_hor_visc.F90#L1258), and [here](https://github.com/NOAA-GFDL/MOM6/blob/b92d763ec91d57d3868162d5d9a24827d2e53674/src/parameterizations/lateral/MOM_hor_visc.F90#L1294). 

In summary, the code is not wrong, but the user should specify `CS%KH_aniso = 2 k_a` (rather than `CS%KH_aniso = k_a`, where the latter is maybe what one would expect after reading the docs).
